osdsmcctl/cli/cli.go

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2019 Huawei Technologies Co., Ltd. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/*
+This module implements a entry into the OpenSDS CLI service.
+
+*/
+
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/opensds/multi-cloud/api/pkg/filters/context"
+	"github.com/opensds/multi-cloud/api/pkg/utils"
+	c "github.com/opensds/multi-cloud/client"
+	"github.com/spf13/cobra"
+	yaml "gopkg.in/yaml.v2"
+)
+
+var (
+	client      *c.Client
+	rootCommand = &cobra.Command{
+		Use:   "osdsmcctl",
```

Comment:
too long, we can use 'gelato' or 'mcctl' or other simple name as the prefix of command.

Reply:
maybe it's better to user unified command name "osdsctl"

Reply:
Use “mcctl“” as the command name